lib/pundit.rb

```diff
@@ -9,6 +9,19 @@
 module Pundit
   class NotAuthorizedError < StandardError
     attr_accessor :query, :record, :policy
+
+    def self.create(options = {})
```

Comment:
Right, for some weird reasons I didn't want to override default error constructor. Changed ;)